src/StateService/Settings.cs

```diff
@@ -1,18 +1,21 @@
 using Microsoft.Extensions.Configuration;
+using System.Linq;
 
 namespace Neo.Plugins.StateService
 {
     internal class Settings
     {
         public string Path { get; }
         public bool FullState { get; }
+        public string[] Validators { get; }
```

Comment:
`Uri` ?

Reply:
`MPTValidators` `MPTDispachers` `MPTVerifiers`
I think it would be better to chose a name that does not cross with dBFT `Validators`

Reply:
> `Uri` ?

Yes. Like Oracle.

Reply:
> `MPTValidators` `MPTDispachers` `MPTVerifiers`
> I think it would be better to chose a name that does not cross with dBFT `Validators`

Let's take `MPTVerifier`.

Reply:
Done.

Reply:
I mean change the type https://github.com/neo-project/neo-modules/pull/470/commits/70e7201a241abbca0492a261520f339fe85da1e7